Sources/ViewInspector/SwiftUI/CustomView.swift

```diff
@@ -72,7 +72,7 @@ internal extension Content {
 @available(iOS 13.0, macOS 10.15, tvOS 13.0, *)
 public extension InspectableView where View: SingleViewContent {
     
-    func view<T>(_ type: T.Type) throws -> InspectableView<ViewType.View<T>> where T: InspectableProtocol {
```

Comment:
This was necessary because previously `ViewType.View<T>` needed to conform to `InspectableProtocol` though that requirement no longer exists.